recipes-bsp/acpi-tables/files/acpi-tables-load

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# This script copies all ACPI tables into /kernel/firmware/acpi
+# You must have configfs enabled for this to work
+
+    for TABLE in `ls /kernel/firmware/acpi/ 2>/dev/null`; do
+	dest_dir=$(basename $TABLE .aml)
+	mkdir /sys/kernel/config/acpi/table/$dest_dir
```

Comment:
Maybe `CONFIGFS_DIR="/sys/..."` and then `mkdir $CONFIGFS_DIR/...` and so on?

Reply:
Good